### PR TITLE
feature/signup-pay-flow 

### PR DESCRIPTION
**Pull request by zhen1asemen1uk**

### CODE REVIEW

File: cypress/support/yll/actions.js [hunk 1]
```diff
@@ -724,11 +744,26 @@ const makePayment = ({email = "", loanName, amount}) => {
                 cy.get('input#paymentAmount').type(amount);
     
                 const bankAccount = account.bankAccounts[Object.keys(account.bankAccounts).slice(-1)];
+
                 cy.contains('Select Bank Account').parent().contains(bankAccount.bankName, {matchCase: false}).click();
                 cy.contains('Payment Timing').parent().contains('Payment Due Now').click();
                 cy.contains('label', 'Notes (Optional)').parent().get('input[placeholder="Extra January Payment"]').type('Cypress automated test payment.');
     
-                cy.contains('button', 'Review Payment Details').click();
+//
```
Comment (MichaelKatic) [inline cypress/support/yll/actions.js:752]: Remove this comment.

File: cypress/support/yll/actions.js [hunk 2]
```diff
@@ -724,11 +744,26 @@ const makePayment = ({email = "", loanName, amount}) => {
...
+                cy.get('[name="paymentMethod"]').click()
+                cy.contains('button', 'Review Payment Details').click({force:true});
+                cy.get('button#mui-5').contains('Confirm Payment of').click()
+                cy.contains('button', 'Close').click();
+                cy.contains("div","Payment Methods").click({force:true})
+                cy.contains("a","Payment Methods").click({force:true})
+//
```
Comment (MichaelKatic) [inline cypress/support/yll/actions.js:759]: Remove this comment and the commented out lines below please.

File: cypress/support/yll/actions.js [hunk 3]
```diff
@@ -576,11 +577,19 @@ const setupPaymentAccount = ({email = ''} = {}) => {
 
             //Verify details popup
             const confirmDetailsModel = cy.contains('Please verify bank details below').parent(); 
-            confirmDetailsModel.within(() => {
-                cy.contains(newBankAccount.routingNumber);
-                cy.contains(newBankAccount.accountNumber);
-                cy.contains(newBankAccount.bankName);
-            })
+
```
Comment (MichaelKatic) [inline cypress/support/yll/actions.js:580]: Remove all commented out code.

**Comment by MichaelKatic** [comment]
@zhen1asemen1uk , please remove all the commented code out. Also, I sent you this [PR](https://github.com/olmsteadproperties/YourLandLoans-Testing/pull/5/files). Make sure you merge it into your branch or manually add the changes now that there are conflicts. Without this you'll continue seeing the issue you encountered last week.

File: cypress/support/yll/actions.js (hunk 1 above)
Comment (zhen1asemen1uk) [reply cypress/support/yll/actions.js:752]: Okay

File: cypress/support/yll/actions.js (hunk 3 above)
Comment (zhen1asemen1uk) [reply cypress/support/yll/actions.js:580]: Okay

**Pull request closed without merging**